agent/src/commands/runBackup.ts

```diff
@@ -20,13 +20,13 @@ export const runBackup: runBackupInterface = async ({ serverId }) => {
   await exec(`screen -S "${serverId}" -p 0 -X stuff "/save-all\r"`);
 
   // create the tar
-  await exec(`cd ../../servers/${serverId} && tar -zcvf ${serverId}.tar.gz .`);
+  await exec(`cd $SERVERS_DIR/${serverId} && tar -zcvf ${serverId}.tar.gz .`);
```

Comment:
Where is this defined?  Shouldn't this be a process.env variable somewhere?

Reply:
This is works the same as the process.env. The SERVER_DIR is added to the system environment at the start time of the Node Js process and can be accessed via code with the process.env or with $SERVERS_DIR in the context of executing command directly with exec. The alternative of this is putting `path.join(process.env.SERVERS_DIR, "/${serverId}")`